testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -417,8 +420,15 @@
 
 GSD = resource_filename(__name__, 'data/example.gsd')
 
-DihedralsArray = resource_filename(__name__, 'data/adk_oplsaa_dihedrals.npy')
-GLYDihedralsArray = resource_filename(__name__, 'data/adk_oplsaa_GLY_dihedrals.npy')
+DihedralArray = resource_filename(__name__, 'data/adk_oplsaa_dihedral.npy')
+DihedralsArray = resource_filename(__name__, 'data/adk_oplsaa_dihedral_list.npy')
+RamaArray = resource_filename(__name__, 'data/adk_oplsaa_rama.npy')
+GLYRamaArray = resource_filename(__name__, 'data/adk_oplsaa_GLY_rama.npy')
+JaninArray = resource_filename(__name__, 'data/adk_oplsaa_janin.npy')
+LYSJaninArray = resource_filename(__name__, 'data/adk_oplsaa_LYS_janin.npy')
+
+Rama_ref = resource_filename(__name__, 'data/rama_ref_data.npy')
+Janin_ref = resource_filename(__name__, 'data/janin_ref_data.npy')
```

Comment:
Needs to go into MDAnalysis (see above).

Reply:
Can you also prepare references for each amino acid separately? I think that this would be more interesting than all sidechains averaged. The peptide backbone is not strongly dependent on the chemical identity of the residue (GLY, PRO are exceptions) but the Janin plots should depend strongly on the nature of the sidechain.